### ISSUE TITLE
1 WETH = ~18 USDC

### ISSUE THREAD

**Issue by salemalem** [kiknaio/Uniswap-swapping-and-add-liquidity-programatically#2]
![image](https://user-images.githubusercontent.com/42411937/226281878-807c9c28-7b19-4b30-b2d2-622940950e42.png)

Debugged fee, state, slippage, everything seems good tho.

**Comment by kiknaio** [comment]
Yep, everything's working except creating a `Trade` object `const trade = await Trade.exactIn(route, amountIn);`

**Comment by salemalem** [comment]
@kiknaio  I don't have any Trade and it's still showing eth price incorrectly.

**Comment by kiknaio** [comment]
Pull the latest changes

**Comment by kiknaio** [comment]
Let me know if it will not work
